src/Services/ProtocolActivationHandler.cs

```diff
@@ -19,14 +19,15 @@ public ProtocolActivationHandler(INavigationService navigationService)
         this._navigationService = navigationService;
     }
 
-    protected override Task HandleInternalAsync(ProtocolActivatedEventArgs args)
+    protected override bool CanHandleInternal(ProtocolActivatedEventArgs args)
     {
-        if (args.Uri.AbsolutePath == SettingsAccountsUri)
-        {
-            _navigationService.DefaultPage = typeof(AccountsViewModel).FullName!;
-            _navigationService.NavigateTo(_navigationService.DefaultPage);
-        }
+        return args.Uri != null && args.Uri.AbsolutePath.Equals(SettingsAccountsUri, StringComparison.OrdinalIgnoreCase);
+    }
 
+    protected override Task HandleInternalAsync(ProtocolActivatedEventArgs args)
+    {
+        _navigationService.DefaultPage = typeof(AccountsViewModel).FullName!;
+        _navigationService.NavigateTo(_navigationService.DefaultPage);

```

Comment:
I know this was existing code, but should we just make it
```suggestion
        _navigationService.NavigateTo(typeof(AccountsViewModel).FullName!);
```